File: Difficulty: Easy/Calculator/calculator.c
//{ Driver Code Starts
// Initial Template for C

// } Driver Code Ends

#include <stdio.h>

int main() {
    // Declare variables for input
    int a, b, operator;
    
    // Read input values: two integers and an operator code
    scanf("%d%d%d", &a, &b, &operator);
    
    // Switch statement to perform the operation based on the operator value
    switch (operator) {
        case 1:
            // If operator is 1, perform addition
            printf("%d", a + b);
            break;
        case 2:
            // If operator is 2, perform subtraction
            printf("%d", a - b);
            break;
        case 3:
            // If operator is 3, perform multiplication
            printf("%d", a * b);
            break;
        default:
            // If operator is not 1, 2, or 3, print "Invalid Input"
            printf("Invalid Input");
    }

    return 0;
}



//{ Driver Code Starts.

// } Driver Code Ends

File: Difficulty: Easy/Calculator/calculator.java
//{ Driver Code Starts
//Initial Template for Java

import java.util.Scanner;

class Main {

    
// } Driver Code Ends

// User function Template for Java

public static void utility(int a, int b, int operator) {
    // Use switch-case to perform the operation based on the operator value
    switch (operator) {
        case 1:  // Addition
            System.out.print(a + b);
            break;
        case 2:  // Subtraction
            System.out.print(a - b);
            break;
        case 3:  // Multiplication
            System.out.print(a * b);
            break;
        default:  // Invalid input for operator
            System.out.print("Invalid Input");
            break;
    }
}



//{ Driver Code Starts.

    public static void main(String[] args)
    {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        while(t-- > 0) {
            int a = scn.nextInt();
            int b = scn.nextInt();
            int operator = scn.nextInt();
            utility(a, b, operator);
            System.out.println();
        }
        scn.close();
    }
}
// } Driver Code Ends